shared/js/media/video_player.js

```diff
@@ -456,9 +456,8 @@ function VideoPlayer(container) {
   });
 
   function formatTime(time) {
-    time = Math.round(time);
     var minutes = Math.floor(time / 60);
-    var seconds = time % 60;
+    var seconds = Math.floor(time % 60);
```

Comment:
By using floor() here instead of round(), you'll have cases where videos of duration 9.99 seconds are displayed as 9 seconds long instead of 10 seconds. I'd suggest that changing MediaUtils.formatDuration() to use round instead of floor might be the better choice.